src/typography/__tests__/Code.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import faker from 'faker'
+import renderer from 'react-test-renderer'
+import { ThemeProvider } from '../../theme'
+import { defaultTheme } from '../../themes'
+import Code from '../src/Code'
+
+describe('Code', () => {
+  it('Should render', () => {
+    expect(() => render(<Code>This is my code</Code>)).not.toThrow()
+  })
+
+  it.each([
+    ['size 300', 300],
+    ['size 400', 400],
+    ['size 500', 500],
+    ['size 600', 600]
+  ])('<Code /> %s renders as expected', (_, size) => {
+    const component = (
+      <ThemeProvider value={defaultTheme}>
+        <Code size={size}>{`Text ${size}`}</Code>
+      </ThemeProvider>
+    )
+    const tree = renderer.create(component).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+
+  it.each([
+    ['size 300', 300],
+    ['size 400', 400],
+    ['size 500', 500],
+    ['size 600', 600]
+  ])('<Code /> %s with minimal appearance specified renders as expected', (_, size) => {
+    const component = (
+      <ThemeProvider value={defaultTheme}>
+        <Code size={size} appearance="minimal">{`Text ${size}`}</Code>
+      </ThemeProvider>
+    )
+    const tree = renderer.create(component).toJSON()
+    expect(tree).toMatchSnapshot()
```

Comment:
Hmmm... I wish we had a better way to test for this, but I know this is coming from one of the other test suites. I'm fine with leaving it as-is and we can revisit it later